net/freeradius/src/opnsense/scripts/Freeradius/setup.sh

```diff
@@ -15,3 +15,7 @@ for FILE in ${RADIUS_FILES}; do
 	touch ${FILE}
 	chmod 700 ${FILE}
 done
+
```

Comment:
we should clear the certificates written here before generating them again, otherwise previously used (now unused) certificates linger in the directory forever

Reply:
good catch, fixed